### ISSUE TITLE
Tab autocomplete IRC names ingame

### ISSUE THREAD

**Issue by WizardCM** [WMCAlliance/BukkitIRCd#54]
Currently, this doesn't work. I'm not actually sure if it'd be possible without adding said players to the tablist.

**Comment by chyyran** [comment]
This might be client-side, at least to a point. Your best bet is with SpoutPlugin and SpoutLauncher :/

**Comment by WizardCM** [comment]
Yeah, I thought it would be. What if we made IRC names show in the tablist?

**Comment by chyyran** [comment]
That would make the tablist pretty cluttered, but you could try that. I don't expect it will have much effect though

**Comment by BlaneyXYZ** [comment]
What about a scoreboard showing online IRC users? and if they are away or not?
